examples/wordle/src/App.tsx

```diff
@@ -174,6 +174,16 @@ function App() {
           <li>An incorrect letter stays black.</li>
         </ul>
       </div>
+      <div className="cs11 ce12">
+        <div className="feedback">
+          <a
+            target="_top"
+            href="https://github.com/miroapp/app-examples/issues"
+            title="Leave feedback"
```

Comment:
The link points to GitHub issues, we should let users know where they are going, or it may be confusing for them when they land on GitHub issues and expect a feedback form instead.